src/templates/staff/assignment.html

```diff
@@ -219,12 +226,201 @@
 </nav>
 <div class="container-fluid">
     <div class="row">
-        <div class="col-md-6 col-lg-8 col-xl-9">
+        {% if is_admin %}
+            <div class="card" style="flex-direction: row; flex: 1;">
+                <div class="card-body">
+                    <div class="container-fluid" style="padding: 0;">
+                        <div class="row">
+                            <div class="col-md-6 col-lg-6 col-xl-4" style="align-self: center;">
+                                <h2 class="card-title">{{ assignment.assignment_id }}</h5>
+                            </div>
+
+                            <div class="col-md-6 col-lg-6 col-xl-8" style="align-self: center; text-align: center;">
+                                <div class="row">
+                                    <div class="col" style="padding-inline: 3px;">
```

Comment:
nit: I'm not sure the `style="padding-inline: 3px;"` is doing for these columns. In Chrome, this property seems to be overwritten. If it's not needed, we should remove them.

Reply:
Let's add a `p-1` class to these columns so that the buttons are not vertically squished together when the screen is small.